### PR TITLE
Agrego paginas guia y modifico changelog

### PR DESCRIPTION
**Pull request by jorgequevedomeli**
## Description

Added ipn_ow communication and a guide to users of this topic

### Issue involved

Fixes #<issue>

### Checklist:
Before sending your contribution, please specify the following: 

- [ ] This request modifies code snippets. 
- [ ] The contribution aligns with the information stated in the repository wiki.

- [x] Contribution incorporates an article not included until now, which must be added to indexes as a new section. 
- [ ] Contribution includes new features. 
- [x] New features were communicated in changelog.
- [ ] Requires translations.

### CODE REVIEW

File: guides/changelog/current-year.en.md [hunk 1]
```diff
@@ -10,6 +10,14 @@ Each release note describes the new changes applying to a version. This changes
 
 - **Documentation updates:** Guides, references and tutorials to help you monetize your business by integrating Mercado Pago.
 
+## November 25, 2019
+
+To meet the highest security standards, we seek to always be updated. Therefore, the old version of **IPN notifications (Instant Payment Notification)** will no longer be available from November 29.
+
+If you are using it, **you must check your version and update it to avoid problems and loss of service.** In case you don't want to do it, you can use other types of notifications that we offer.
+
+You can find all the necessary information in the [migration guide](https://www.mercadopago.com.br/developers/en/guides/changelog/communications).
```
Comment (Juanii95) [inline guides/changelog/current-year.en.md]:
```suggestion
You can find all the necessary information in the [migration guide](https://www.mercadopago.com.br/developers/en/guides/changelog/communications/ipn-ow-guide). 
```

File: guides/changelog/current-year.es.md [hunk 2]
```diff
@@ -10,6 +10,14 @@ Cada release note describe los cambios que aplican a una versión. Estos cambios
 
 - **Actualizaciones en la documentación:** Guías, referencias y tutoriales para ayudarte a monetizar tu negocio integrando Mercado Pago.
 
+## 25 de Noviembre de 2019
+
+Para cumplir con los más altos estándares de seguridad, buscamos siempre estar actualizados. Por eso, la versión antigua de **notificaciones IPN (Instant Payment Notification)** ya no estará disponible a partir del 9 de diciembre.
+
+Si la estás usando, **tienes que revisar tu versión y actualizarla para evitar problemas y pérdidas de servicio.** En caso de no querer hacerlo, puedes usar otro tipo de notificaciones que ofrecemos.
+
+Puedes encontrar toda la información necesaria en la [guía de la migración](https://www.mercadopago.com.ar/developers/es/guides/changelog/communications).
```
Comment (Juanii95) [inline guides/changelog/current-year.es.md]:
```suggestion
Puedes encontrar toda la información necesaria en la [guía de la migración](https://www.mercadopago.com.ar/developers/es/guides/changelog/communications/ipn-ow-guide).
```

File: guides/changelog/current-year.pt.md [hunk 3]
```diff
@@ -10,6 +10,14 @@ Cada release note descreve as alterações que se aplicam a uma versão. Essas a
 
 - **Atualizações na documentação:** Guias, referências e tutoriais para te ajudar a monetizar seu negócio integrando Mercado Pago.
 
+## 24 de novembro de 2019
+
+Para atender aos mais altos padrões de segurança, buscamos estar sempre atualizados. Portanto, a versão antiga das **notificações IPN (Notificação de pagamento instantâneo)** não estará mais disponível a partir do dia 9 de dezembro.
+
+Se a estiver usando, **você deve revisar sua versão e atualizá-la para evitar problemas e perda de serviço.** Caso você não queira fazer isso, pode usar outro tipo de notificação que oferecemos. 
+
+Confira todas as informações necessárias no [manual da migração](https://www.mercadopago.com.br/developers/pt/guides/changelog/communications).
```
Comment (Juanii95) [inline guides/changelog/current-year.pt.md]:
```suggestion
Confira todas as informações necessárias no [manual da migração](https://www.mercadopago.com.br/developers/pt/guides/changelog/communications/ipn-ow-guide).
```

File: guides/changelog/current-year.en.md (hunk 1 above)
Comment (jorgequevedomeli) [reply guides/changelog/current-year.en.md]: done

File: guides/changelog/current-year.es.md (hunk 2 above)
Comment (jorgequevedomeli) [reply guides/changelog/current-year.es.md]: done

File: guides/changelog/current-year.pt.md (hunk 3 above)
Comment (jorgequevedomeli) [reply guides/changelog/current-year.pt.md]: done

**Pull request merged**